### ISSUE TITLE
Rewarded on iOS displayed only once

### ISSUE THREAD

**Issue by erossini** [marcojak/MauiMTAdmob#69]
Hi,
I read posts with the same issue, and I was thinking that maybe you already know how to fix it. In my app, I display a reward advertise and it is displayed. The second time the advertisement is not displayed. 

I tried to call again this code

```
CrossMauiMTAdmob.Current.LoadRewarded(Constants.Reward1);
```

and then I call

```
CrossMauiMTAdmob.Current.ShowRewarded();
```

between the first code and the second could pass at least a few seconds. The second time, the advertisement is not displayed but there is no events.

Do you know how can I fix it?
Thank you in advance,
Enrico

**Comment by marcojak** [comment]
Hi Enrico,

Do you have the same issue with the sample app?

If the issue is there as well, it might be a problem with the plugin. If
so, I'll check and in case update the plugin with the fix.

**Comment by erossini** [comment]
Honestrly, I haven't checked.

**Comment by erossini** [comment]
Hi Marco,
I tried to run your sample project and I have a few issues. So, I created a new one and you can see it on this [repo](https://github.com/erossini/MMTAdmobSample). I have the same issue with your project on a real device.

**Comment by erossini** [comment]
Did you have a chance to look at this code? @marcojak thank you in advance

**Comment by marcojak** [comment]
Yes, I can see the same issue on the sample code...I'm investigating it and I hope to fix it asap

**Comment by erossini** [comment]
Thank you so much!
Enrico

**Comment by marcojak** [comment]
Found the issue and fixed it.
I'm planning to release the new version in the next few days with this fix and possibly with few other improvements.

**Comment by erossini** [comment]
Perfect! Thank you!

**Comment by marcojak** [comment]
Just published version 1.3.3.
Closing for now, but feel free to open a new one in case of issues

**Comment by erossini** [comment]
Thank you so much/ I'll try it now
